helm-chart-sources/pulsar/values.yaml

```diff
@@ -768,8 +768,7 @@ broker:
   # https://github.com/datastax/kop
   kafkaOnPulsarEnabled: false
   kafkaOnPulsar:
-    kafkaListeners: "PLAINTEXT://localhost:9092"
-    kafkaAdvertisedListeners: "PLAINTEXT://localhost:9092"
+    kafkaListeners: "PLAINTEXT://0.0.0.0:9092"
```

Comment:
this kafkaOnPulsar section could be dropped at all
@michaeljmarshall 

because there is already a way to pass custom configurations to the broker and to the proxy